rancher/v1.6/en/rancher-services/storage-service/rancher-nfs/index.md

```diff
@@ -55,7 +55,8 @@ In order to launch Rancher NFS, you will need to specify the following:
 * **NFS Server**: NFS server ip address or hostname.
 * **Export Base Directory**: Exported shared directory on the NFS server.
 * **NFS Version**: The NFS version to use, current used version is 4.
-* **Mount Option**: Comma delimited list of default mount options, for example: 'proto=udp'. Do not specify `nfsvers` option, it will be ignored.
+* **Mount Options**: Comma delimited list of default mount options, for example: 'proto=udp'. Do not specify `nfsvers` option, it will be ignored.
+* **On Remove**: On removal of Rancher NFS volume, should the underlying data be retained or purged. Default is `purge`.
```

Comment:
Can we add in what are the 2 options we support? `purge` and  `retain`? Also, can you note that it's only as of Rancher v1.6.6+ (that's where the NFS version will be made available).